.storybook/static/core-theme.css

```diff
@@ -103,6 +103,8 @@
 --color-warn-lighter: #fce28f;
 --color-warn-lightest: #fef9e9;
 --color-visited: #4c2c92;
+--font-body: 'Open Sans', Helvetica, sans-serif;
+--font-heading: 'Open Sans', Helvetica, sans-serif;
```

Comment:
I created these `--font-*` vars to make it easier to switch between themes on the reset. After I made them, I noticed we also have `--typography-` variables defined and I was wondering if we should standardize behind one? 

Personally, seeing all the "font" vars up top made me think we had outages because I didn't realize there were additional vars with the "typography" prefix towards the bottom.

Reply:
I do think they refer to different contexts, but it's not super clear. The `typography` prefixed ones are used in the typography-related classes like headings and `ds-text-body`, whereas if we're applying this in a reset that includes other styles not related to typography, it wouldn't necessarily fit to have the `typography` prefix. Just trying to make sense of what's there already; I'm not prescribing anything. It also looks like `ds-text` is a duplicate of `ds-text-body`. I think `ds-text` is the older one and out of fashion since the `ds-text-heading*` classes came along.

Reply:
**A typeface/typography** is a set of design features for letters and other characters, like the presence or lack of a serif, the letters’ weight and balance, spacing and the height difference between upper and lowercase letters. So think Times New Roman  
  
**A font** is the variation in weight and size of a typeface. So when a typeface is, bold, italic, condensed, size or any other variable, that’s a font.  
  
So there is a difference and it is also definitely nuanced

Reply:
I'd recommend something like `--typography-body` or `--typeface-body` and using `--font` for the items that deal with sizing, weight, etc...